user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -404,28 +411,28 @@
               </app-live-workshop>
 
 
-              <app-live-workshop 
-                title="Geico: Weather API" 
-                descrip="Finding a data source is easy, but harnessing that data into a valuable perspective
+              <app-live-workshop
+                title="Geico: Weather API"
+                description="Finding a data source is easy, but harnessing that data into a valuable perspective
                 is an art. Is today a good day for a Sunday drive? Do I need an umbrella in
                 my car or a kayak? How do you dive deep on data but maintain a lightweight UI
                 to provide context without concern? Join the GEICO Mobile team in their challenge
                 to craft an engaging and informative fusion of data sources to provide value to
-                their customers and the products they insure." 
-                date="Saturday" 
-                time="3:00PM-3:45pm" 
+                their customers and the products they insure."
+                date="Saturday"
+                time="3:00PM-3:45pm"
                 link="http://www.google.com"
                 presenter1img="../../../assets/images/workshop_leaders/DavidKing.jpg"
                 presenter1Name="David King"
                 collapseID="geico"
                 >
               </app-live-workshop>
 
-              <app-live-workshop 
-                title="Team Building!" 
-                descrip="Don’t have a team? Fear not! Come to our Team Building workshop to connect with Hackers and to find the people you will build your winning project with!" 
-                date="88/88/8888" 
-                time="88:88PM" 
+              <app-live-workshop
+                title="Team Building!"
+                description="Don’t have a team? Fear not! Come to our Team Building workshop to connect with Hackers and to find the people you will build your winning project with!"
+                date="88/88/8888"
+                time="88:88PM"
                 link="http://www.google.com"
                 presenter1img="../../../assets/images/workshop_leaders/RahulWalia.jpg"
                 presenter1Name="Rahul Walia"
```

Comment:
Also right below this line (if you expand it), there is still an AccuWeather image!